HACKER/HACKER/Sources/ViewControllers/MainViewController/MainProfileViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  MainProfileViewController.swift
+//  HACKER
+//
+//  Created by 김지수 on 2022/03/18.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+// MARK: - MainProfileViewController
+class MainProfileViewController: UIViewController {
+  
+  // MARK: - Components
+  let backButton = UIButton()
+  let userCharacterImage = UIImageView()
+  let userHairImage = UIImageView()
+  let userNicknameLabel = UILabel()
+  let userGithubNameLabel = UILabel()
+  let hairNumLabel = UILabel()
+  
+  // MARK: - LifeCycle
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.view.backgroundColor = .hackerWhite
+    self.navigationController?.navigationBar.isHidden = true
+    layout()
+  }
+}
+// MARK: - Extensions
+extension MainProfileViewController {
+  func layout() {
+    layoutBackButton()
+    layoutUserCharacterImage()
+    layoutUserHairImage()
+    layoutUserNickNameLabel()
+    layoutUserGithubNameLabel()
+    layoutHairNumLabel()
+  }
+  func layoutBackButton() {
+    view.add(backButton) {
+      $0.setImage(UIImage(named: "backButtonIcon"), for: .normal)
+      $0.addTarget(self, action: #selector(self.backButtonTapped), for: .touchUpInside)
+      $0.snp.makeConstraints {
+        $0.top.equalTo(self.view.safeAreaLayoutGuide.snp.top).offset(16)
+        $0.leading.equalToSuperview().offset(24)
+        $0.width.equalTo(40)
+        $0.height.equalTo(40)
+      }
```

Comment:
코드 쓰는 스타일을 이렇게 쓰기로 협의했다면 쓰는 게 맞지만
아래처럼 해보면 더 간단하게 쓸 수도 있어
```suggestion
      $0.snp.makeConstraints {
        $0.top.equalTo(self.view.safeAreaLayoutGuide).offset(16)
        $0.leading.equalToSuperview().offset(24)
        $0.width.height.equalTo(40)
      }
```